docs/sources/clients/promtail/configuration.md

```diff
@@ -60,6 +60,11 @@ ${VAR:default_value}
 
 Where default_value is the value to use if the environment variable is undefined.
 
+**Note**: With `expand-env=true` the configuration will first run through
+[envsubst](https://linux.die.net/man/1/envsubst) which will replace double
```

Comment:
```suggestion
[envsubst](https://pkg.go.dev/github.com/drone/envsubst) which will replace double
```
would recommend to link the actual package we use to avoid missing any __real__ behaviour of `envsubst`.